tests/Makefile

```diff
@@ -1,12 +1,13 @@
-PATH := ../deps/luajit/usr/local/bin:$(PATH)
+PATH := ../deps/luajit/usr/local/bin:../../deps/luajit/usr/local/bin:$(PATH)
```

Comment:
The regression tests are one level deeper in the directory tree, the CI servers don't have luajit in their paths (or set up at all outside of our builds), and our convention thus far is to use relative paths. It's not pretty, but none of the options I see are.